core-test/src/test/scala/org/bitcoins/core/protocol/script/ScriptPubKeyTest.scala

```diff
@@ -41,7 +41,7 @@ class ScriptPubKeyTest extends BitcoinSUnitTest {
   }
 
   it must "construct valid witness spk v1 for taproot" in {
-    val pubKey = CryptoGenerators.schnorrPublicKey.sample.get
+    val pubKey = CryptoGenerators.schnorrPublicKey.sample.get.toXOnly
```

Comment:
Should do `CryptoGenerators.xOnlyPubKey.sampleSome`, as this would only test valid `TaprootScriptPubKey`s with known parity